eddymotion/tests/test_estimator.py

```diff
@@ -20,4 +20,54 @@
 #
 #     https://www.nipreps.org/community/licensing/
 #
-"""Unit tests exercising the estimator."""
\ No newline at end of file
+"""Unit tests exercising the estimator."""
+from tempfile import TemporaryDirectory
+from pkg_resources import resource_filename as pkg_fn
+from pathlib import Path
+
+import nitransforms as nt
+import numpy as np
+import random
+
+from nibabel.eulerangles import euler2mat
+from nibabel.affines import from_matvec
+from nipype.interfaces.ants.registration import Registration
+
+
+def test_ANTs_config_b0(pkg_datadir, tmpdir):
+    """Check that the registration parameters for b=0
+    gives a good estimate of known affine"""
+
+    tmpdir.chdir()
+    for i in range(100):
+        # Generate test transfrom with random small parameters
+        x = random.randint(0, 1000)
+        y = random.randint(0, 1000)
+        z = random.randint(0, 1000)
+        a = random.randint(0, 20)
+        b = random.randint(0, 20)
+        c = random.randint(0, 20)
```

Comment:
Let's replace this with parameterized tests. Basically, that means pytest can do the exploration of the parameters for you.

Example: https://github.com/nipreps/sdcflows/blob/c802b4deff69348a8dd1470c92ba18ab9971f19c/sdcflows/tests/test_transform.py#L77-L80

```suggestion
```